languages.toml

```diff
@@ -1249,3 +1249,16 @@ language-server = { command = "vala-language-server" }
 [[grammar]]
 name = "vala"
 source = { git = "https://github.com/vala-lang/tree-sitter-vala", rev = "c9eea93ba2ec4ec1485392db11945819779745b3" }
+
+[[language]]
+name = "hare"
+scope = "source.hare"
+injection-regex = "hare"
+file-types = ["ha"]
+roots = []
+comment-token = "//"
+indent = { tab-width = 4, unit = "\t" }
+
+[[grammar]]
+name = "hare"
+source = { git = "https://git.sr.ht/~ecmma/tree-sitter-hare", rev = "bc26a6a949f2e0d98b7bfc437d459b250900a165" }
```

Comment:
A note for onlookers: this is the first grammar not hosted on GitHub. I don't think we have anything hardcoded to GitHub anywhere though so this shouldn't be a problem.